.github/workflows/main.yaml

```diff
@@ -157,7 +157,7 @@ jobs:
 
       - name: Clone
         shell: bash -l {0}
-        run: git clone https://github.com/fsspec/${{ matrix.FRIEND }}
+        run: git clone -b better-double-asterisk-support https://github.com/john-jam/${{ matrix.FRIEND }}
```

Comment:
Note for later: remove this (decide to merge the friends repo branches or not)

Reply:
Noted. I'd normally be inclined to merge this first, then follow up with the `s3fs` and `gcsfs` PRs in quick succession.

Reply:
Note: reverted